provider/resources.go

```diff
@@ -1326,22 +1335,24 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_default_vpc":  {Tok: awsResource(ec2Mod, "DefaultVpc")},
 			"aws_vpc":          {Tok: awsResource(ec2Mod, "Vpc")},
 			"aws_vpc_endpoint": {Tok: awsResource(ec2Mod, "VpcEndpoint")},
-			"aws_vpc_endpoint_connection_notification":   {Tok: awsResource(ec2Mod, "VpcEndpointConnectionNotification")},
-			"aws_vpc_endpoint_route_table_association":   {Tok: awsResource(ec2Mod, "VpcEndpointRouteTableAssociation")},
-			"aws_vpc_endpoint_service":                   {Tok: awsResource(ec2Mod, "VpcEndpointService")},
-			"aws_vpc_endpoint_service_allowed_principal": {Tok: awsResource(ec2Mod, "VpcEndpointServiceAllowedPrinciple")},
-			"aws_vpc_endpoint_subnet_association":        {Tok: awsResource(ec2Mod, "VpcEndpointSubnetAssociation")},
-			"aws_vpc_ipv4_cidr_block_association":        {Tok: awsResource(ec2Mod, "VpcIpv4CidrBlockAssociation")},
-			"aws_vpn_connection":                         {Tok: awsResource(ec2Mod, "VpnConnection")},
-			"aws_vpn_connection_route":                   {Tok: awsResource(ec2Mod, "VpnConnectionRoute")},
-			"aws_vpn_gateway":                            {Tok: awsResource(ec2Mod, "VpnGateway")},
-			"aws_vpn_gateway_attachment":                 {Tok: awsResource(ec2Mod, "VpnGatewayAttachment")},
-			"aws_vpn_gateway_route_propagation":          {Tok: awsResource(ec2Mod, "VpnGatewayRoutePropagation")},
-			"aws_ec2_traffic_mirror_filter":              {Tok: awsResource(ec2Mod, "TrafficMirrorFilter")},
-			"aws_ec2_traffic_mirror_filter_rule":         {Tok: awsResource(ec2Mod, "TrafficMirrorFilterRule")},
-			"aws_ec2_traffic_mirror_session":             {Tok: awsResource(ec2Mod, "TrafficMirrorSession")},
-			"aws_ec2_traffic_mirror_target":              {Tok: awsResource(ec2Mod, "TrafficMirrorTarget")},
-			"aws_ec2_availability_zone_group":            {Tok: awsResource(ec2Mod, "AvailabilityZoneGroup")},
+			"aws_vpc_endpoint_connection_notification":    {Tok: awsResource(ec2Mod, "VpcEndpointConnectionNotification")},
+			"aws_vpc_endpoint_route_table_association":    {Tok: awsResource(ec2Mod, "VpcEndpointRouteTableAssociation")},
+			"aws_vpc_endpoint_service":                    {Tok: awsResource(ec2Mod, "VpcEndpointService")},
+			"aws_vpc_endpoint_service_allowed_principal":  {Tok: awsResource(ec2Mod, "VpcEndpointServiceAllowedPrinciple")},
```

Comment:
It's been this way for a bit apparently but this should be `VpcEndpointServiceAllowedPrincipal` to map correctly, no? Not sure how hard it is to fix after the fact

Reply:
I am going to say we don't need to fix this in this release